website/docs/docs/collaborate/govern/model-versions.md

```diff
@@ -6,39 +6,140 @@ description: "Version models to help with lifecycle management"
 
 ## Related documentation
 - [`versions`](resource-properties/versions)
-- [`latest_version`](resource-properties/latest-version)
+- [`latest_version`](resource-properties/latest_version)
 - [`include` & `exclude`](resource-properties/include-exclude)
+- [`ref` with `version` argument](ref#versioned-ref)
 
 ## Why version a model?
 
-If a model defines a ["contract"](model-contracts) (a set of guarantees for its structure), it's also possible to change that model's contract in a way that "breaks" the previous set of parameters.
+If a model defines a ["contract"](model-contracts) (a set of guarantees for its structure), it's also possible to change that model's structure in a way that breaks the previous set of guarantees. This could be as obvious as removing or renaming a column, or more subtle, like changing its data type or nullability.
+
+One approach is to force every model consumer to immediately handle the breaking change as soon as it's deployed to production. This is actually the appropriate answer at many smaller organizations, or while rapidly iterating on a not-yet-mature set of data models. But it doesn’t scale well beyond that.
+
+Instead, for mature models at larger organizations, powering queries inside & outside dbt, the model owner can use **model versions** to:
+- Test "prerelease" changes (in production, in downstream systems)
+- Bump the latest version, to be used as the canonical source of truth
+- Offer a migration window off the "old" version
+
+During that migration window, anywhere that model is being used downstream, it can continue to be referenced at a specific version.
+
+In the future, dbt will also offer first-class support for **deprecating models** ([dbt-core#7433](https://github.com/dbt-labs/dbt-core/issues/7433)). Taken together, model versions and deprecation offer a pathway for model producers to _sunset_ old models, and consumers the time to _migrate_ across breaking changes. It's a way of managing change across an organization: develop a new version, bump the latest, slate the old version for deprecation, update downstream references, and then remove the old version.
+
+There is a real trade-off that exists here—the cost to frequently migrate downstream code, and the cost (and clutter) of materializing multiple versions of a model in the data warehouse. Model versions do not make that problem go away, but by setting a deprecation date, and communicating a clear window for consumers to gracefully migrate off old versions, they put a known boundary on the cost of that migration.
+
+## When should you version a model?
+
+By enforcing a model's contract, dbt can help you catch unintended changes to column names and data types that could cause a big headache for downstream queriers. These changes, when made intentionally, would require a new model version. But many changes are not breaking, and don't require a new version—such as adding a new column, or fixing a bug in an existing column's calculation.
```

Comment:
Good clarifying question. If you make a breaking contract change, dbt will raise an error during CI — and you'd need to "merge on red" (you always _can_ do it). https://docs.getdbt.com/reference/resource-configs/contract#detecting-breaking-changes
```
  Consider making an additive (non-breaking) change instead, if possible.
  Otherwise, create a new model version: https://docs.getdbt.com/docs/collaborate/govern/model-versions
```